firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1619,6 +1619,49 @@ extension String {
             value: "Skip",
             comment: "Describes an action on some of the Onboarding screen, including the wallpaper onboarding screen. This string will be on a button so user can skip that onboarding page.")
 
+        public struct TermsOfService {
+            public static let Title = MZLocalizedString(
+                key: "", // Onboarding.TermsOfService.Title.v135
+                tableName: "Onboarding",
+                value: "Welcome to %@",
+                comment: "Title for the Terms of Service screen in the onboarding process. Placeholder is for app name.")
+            public static let AgreementButtonTitle = MZLocalizedString(
+                key: "", // Onboarding.TermsOfService.AgreementButtonTitle.v135
+                tableName: "Onboarding",
+                value: "Agree and continue",
+                comment: "Title for the confirmation button for Terms of Service agreement, in the Terms of Service screen.")
+            public static let TermsOfServiceAgreement = MZLocalizedString(
+                key: "", // Onboarding.TermsOfService.TermsOfServiceAgreement.v135
+                tableName: "Onboarding",
+                value: "By continuing, you agree to the %@",
+                comment: "Agreement text for Terms of Service in the Terms of Service screen. Placeholder is for the Terms of Service link button that redirect the user to the Terms of Service page")
+            public static let PrivacyNoticeAgreement = MZLocalizedString(
+                key: "", // Onboarding.TermsOfService.PrivacyNoticeAgreement.v135
+                tableName: "Onboarding",
+                value: "%@ cares about your privacy. Read more in our %@",
+                comment: "Agreement text for Privacy Notice in the Terms of Service screen. First placeholder is for the app name. The second placeholder is for the Privacy Notice link button that redirect the user to the Privacy Notice page")
+            public static let ManagePreferenceAgreement = MZLocalizedString(
+                key: "", // Onboarding.TermsOfService.ManagePreferenceAgreement.v135
+                tableName: "Onboarding",
+                value: "To help improve the browser, %@ sends diagnostic and interaction data to Mozilla. %@",
+                comment: "Agreement text for sends diagnostic and interaction data to Mozilla in the Terms of Service screen. First placeholder is for the app name. The last placeholder is for for the Manage link button which redirect the user to another screen in order to manage the data collection preferences.")
+            public static let TermsOfServiceLink = MZLocalizedString(
+                key: "", // Onboarding.TermsOfService.TermsOfServiceLink.v135
+                tableName: "Onboarding",
+                value: "%@ Terms of Service.",
```

Comment:
I have added that `.` there to be also part of the link, as in the design.
<img width="677" alt="Screenshot 2024-11-22 at 09 20 17" src="https://github.com/user-attachments/assets/9445cb7e-081f-4849-bd29-281047f28e94">

Reply:
@thatswinnie

Reply:
@dicarobinho ok, let's leave it like this until we have the final strings. However I think leaving the `.` like this will cause issues with translations.